apps/core/src/migration/service/convert-monitor-to-app-definition.ts

```diff
@@ -316,6 +331,81 @@ const convertProperties = (
   return getStaticProperties(widgetType);
 };
 
+const convertKpiOrTimelineWidget = (
+  monitorWidget: MonitorWidget,
+  monitorTitle: string,
+): DashboardWidget[] => {
+  // For each metric in a monitor widget, create an application widget
+  if (monitorWidget.metrics) {
+    const maxXCoord = monitorWidget.x + monitorWidget.width - 1;
```

Comment:
why the `-1`?

Reply:
the max x coord is going to be the original X plus the width minus the width of an individual widget (which is 1)